doc/contrib/index.rst

```diff
@@ -0,0 +1,101 @@
+.. _contribindex:
+
+Contributing to Arbor
+=====================
+
+First off, thank you for considering contributing to Arbor! It’s people
+like you that make Arbor such a great tool. Feel welcome and read the
+following sections in order to know how to ask questions and how to work
+on something.
+
+.. _contribindex-types:
+
+Types of contributions
+----------------------
+
+There are many ways to contribute: writing tutorials or blog posts,
+improving the documentation, helping other users, submitting bug reports
+and feature requests or writing code which can be incorporated into
+Arbor itself.
+
+If you come across a bug in Arbor, please `file an issue on the GitHub
+issue tracker <https://github.com/arbor-sim/arbor/issues/new>`__. You
+can attach files to issues. Data and bits of code that help to reproduce
+the issue are very welcome; they’ll make it easier for Arbor developers
+to debug and fix your issue.
+
+The source for `the documentation <https://arbor.readthedocs.io>`__ is
+found in the ``/doc`` subdirectory, and you can add yours or change existing
+documentation in the same way you would contribute code, please see :ref:`contribpr`.
+Some more details on how to write documentation can be found in :ref:`contribdoc`
+
+Please use the `Arbor Discussions <https://github.com/arbor-sim/arbor/discussions>`__
+page for support questions.
+
+.. _contribindex-github:
+
+Github
+------
+
+Arbor uses Git for version control and Github to host its code
+repository and nearly all of its infrastructure.
+
+-  If you’re not familiar with Git or Github, please have at look at
+   `this introduction <https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/set-up-git>`__.
+-  Make sure you have a `GitHub
+   account <https://github.com/signup/free>`__.
+
+.. _contribindex-discuss:
+
+Start a discussion
+~~~~~~~~~~~~~~~~~~
+
+You can use the `Arbor
+Discussions <https://github.com/arbor-sim/arbor/discussions>`__ to help
+other users of Arbor, or get help with you own simulations. Also, we’re
+eager to discover what you’re using Arbor for, so don’t hesitate to
+share your Arbor simulations or publications!
+
+.. _contribindex-fileissue:
+
+Filing an issue
+~~~~~~~~~~~~~~~
+
+If you have found a bug or problem in Arbor, or want to request a
+feature, you can use our `issue
+tracker <https://github.com/arbor-sim/arbor/issues>`__. If you issue is
+not yet filed in the issue tracker, please do so and describe the
+problem, bug or feature as best you can. You can add supporting data,
+code or documents to help make your point.
+
+.. _contribindex-solveissue:
+
+Help solving an Issue
+~~~~~~~~~~~~~~~~~~~~~
+
+If you want to help solve an issue, that’s great! Issue labels can help
```

Comment:
I prefer to avoid the very familiar phrasing like `, that's great!`. I don't think it strikes quite the right tone for the documentation.

`You are welcome to help address issues, which might involve answering questions or contributing code patches that fix bugs and implement new features.`

Reply:
I certainly don't want to overdo it, but I like it if writers are not afraid to show some emotion and character.

In addition, this style is quite common in user-oriented documentation/communication these days I think.